api/v1/models/UniversalTrackingItem.js

```diff
@@ -0,0 +1,17 @@
+var Model = require('./Model');
+
+var UniversalTrackingItem = Model.extend({
+    tableName: 'event_tracking',
+    idAttribute: 'id',
+    hasTimestamps: ['created'],
+    validations: {
+        name: ['required', 'string', 'maxLength:255'],
+        duration: ['required', 'naturalNonZero']
```

Comment:
Is the duration in seconds? Milliseconds? Whatever it is, make sure it's in the docs.